clients/aws-sdk-bedrock-runtime/src/aws_sdk_bedrock_runtime/client.py

```diff
@@ -85,16 +91,23 @@ async def apply_guardrail(
         """
         The action to apply a guardrail.
 
-        For troubleshooting some of the common errors you might encounter when using the
-        ``ApplyGuardrail`` API, see `Troubleshooting Amazon Bedrock API Error Codes <https://docs.aws.amazon.com/bedrock/latest/userguide/troubleshooting-api-error-codes.html>`_
+        For troubleshooting some of the common errors you might encounter when
+        using the `ApplyGuardrail` API, see [Troubleshooting Amazon Bedrock API
+        Error
+        Codes](https://docs.aws.amazon.com/bedrock/latest/userguide/troubleshooting-api-error-codes.html)
```

Comment:
This formatting is a bit weird. Is this expected. I do see this renders properly in the IDE so not too worried. But it does read weird in the Python file.